pkg/ds/farm.go

```diff
@@ -564,6 +565,25 @@ func (dsf *Farm) spawnDaemonSet(
 
 	desiresCh := ds.WatchDesires(quitSpawnCh, updatedCh, deletedCh)
 
+	go func() {
+		aggregateHealth := replication.AggregateHealth(ds.PodID(), *dsf.healthChecker)
```

Comment:
Finally, a third consideration: Previously, the number of active watches accounted for by daemon sets is equal to the number of daemon sets currently doing replications.

With this, the number of active watches increases to the number of daemon sets, at all times, whether they be replicating or not. Does this hurt performance? If so, what do we do about it?